src/components/connect/ConnectSelect.vue

```diff
@@ -180,19 +176,23 @@ export default {
   opacity: 0;
 }
 .slideUp-leave-to {
-  /*transform: translateY(-100%);*/
+  transform-origin: top;
   transform: scaleY(0);
 }
-@media screen and (max-width: 1150px) {
-  /*Tablet View*/
-  #selector {
-    max-width: 510px;
-    width: 100%;
-  }
+
+.slideUp-leave-active * {
+  opacity: 0;
+  visibility: hidden;
+}
+
+@media screen and (min-width: $desktop) {
   .select {
-    width: 35%;
-    max-width: 200px;
-    min-width: 130px;
+    width: calc(25% - 0.8em);
   }
 }
+
+#container {
+  // margin-left: -0.8em;
```

Comment:
Was this comment left here purposefully?

Reply:
I don't see it in my files? I think I deleted it in a later commit.

Reply:
Ok. Sounds good then.

Reply:
I know you're trying to get this out as soon as possible, so don't worry about it. But in the future I would ask you to edit commits.